Geometry/VeryForwardGeometryBuilder/interface/DetGeomDescBuilder.h

```diff
@@ -0,0 +1,14 @@
+#ifndef Geometry_VeryForwardGeometryBuilder_DetGeomDescBuilder
+#define Geometry_VeryForwardGeometryBuilder_DetGeomDescBuilder
+
+#include "Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h"
+#include "DetectorDescription/DDCMS/interface/DDCompactView.h"
+
+/*
+ * Generic function to build geo (tree of DetGeomDesc) from compact view.
+ */
+namespace DetGeomDescBuilder {
```

Comment:
Convention for namespaces is to use lower case letters (or at least start with one).

Reply:
ha yes thanks will fix this